extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+class EmbeddableDiscussionsController extends WikiaApiController {
```

Comment:
I'll remove it. I originally added this as it will be needed for JPN-383
(Community page: as a user, I want to see the most recent discussions posts in my community), but I'll just re-add when I get to that task.